index.d.ts

```diff
@@ -10,9 +16,18 @@ declare module "effection" {
     resume(result: T): void;
     throw(error: Error): void;
     halt(reason?: any): void;
+    send(message: any): void;
```

Comment:
We could limit the message to JSON-like objects, not sure if sending complex objects should be allowed. If we go down the path of messages being transported across different protocols, then sending complex objects should not be allowed.